src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,75 @@
+package mate.academy.model;
+
+import java.util.Arrays;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String login;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+    }
+
+    public User(String login, String password) {
+        this.login = login;
+        this.password = password;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public User setId(Long id) {
+        this.id = id;
+        return this;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public User setLogin(String login) {
+        this.login = login;
+        return this;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public User setPassword(String password) {
+        this.password = password;
+        return this;
+    }
+
+    public byte[] getSalt() {
+        return salt;
+    }
+
+    public User setSalt(byte[] salt) {
+        this.salt = salt;
+        return this;
+    }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
+                + ", salt=" + Arrays.toString(salt)
```

Comment:
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/20326858/dd3115c3-6d6b-430b-9440-f9bd03c4a2c8)

Reply:
I added this for testing - I forgot to remove it. Thank you!